digestparser/json_output.py

```diff
@@ -57,12 +57,13 @@ def image_size(info):
     # only return size if it is non-empty
     if size:
         return size
+    return None
```

Comment:
this is implicit, but no harm in making it explicit

Reply:
`proofreader` output suggested to add it to this one